generate_schema/generate_provider_schema.py

```diff
@@ -0,0 +1,123 @@
+"""
+generate_provider_schema.py
+
+When run, makes standalone schemas for the MDS provider API.
+"""
+
+import json
+import requests
+
+
+def get_point_schema():
+    """
+    Get the canonical schema for a GeoJSON point.
+    """
+    p = requests.get("http://geojson.org/schema/Point.json")
+    point = p.json()
+    # Modify some metadata
+    point.pop("$schema")
+    point["$id"] = "#/definitions/Point"
+    return point
+
+def get_feature_schema():
+    """
+    Get the canonical schema for a GeoJSON Feature,
+    and make the following modifications to match the MDS spec:
+
+    1. Only allow GeoJSON Point features.
+    2. Those Point features *must* include a `timestamp` number property.
+    """
+    # Get the canonical Feature schema
+    f = requests.get("http://geojson.org/schema/Feature.json")
+    feature = f.json()
+    # Modify some metadata
+    feature.pop("$schema")
+    feature["$id"] = "#/definitions/FeatureMDS"
```

Comment:
I wonder if we should call these something like `PointFeatureMDS` and `RouteFeatureCollectionMDS`, in order to leave the door open for other kinds of features and feature collections.

Reply:
Good idea

Reply:
Changes incoming, just turning it into a bit of a larger refactor for the future proofing